posts/symposium-science-cluster.qmd

```diff
@@ -0,0 +1,189 @@
+    of the work that is being presented.
+
+## Unmet expectations
+
+Considering the emphasis on the overall data science lifecycle, I had
+made a few assumptions about the symposium that were not met:
+
+-   *There were very few topics or sessions on data engineering,
+    infrastructure, and management.* I felt this was a bit of a lost
+    opportunity for the symposium, since part of it's stated aim was
+    barely included in the program. It highlights to me, and reinforces
+    my own personal experiences, that these topics are just not well
+    known about, the awareness of them is nearly non-existent, and the
+    value of them is very underappreciated. This is a major issue, since
+    these are the foundational aspects of data science and research, and
+    without them, the rest of the data science lifecycle is not
+    possible.
+
+-   *A lot of presentations and topics were on data in general (or
+    rather data that was collected) and on the analytics.* This is
+    fairly common in other conferences as well. This relates to the
+    previous point, that so often, we in research focus on the data
+    collection and the analytics, but we don't focus on the data
+    engineering and infrastructure that is needed to support the data
+    collection and analytics. Even within grant applications, we write
+    about what data we will collect and how we'll analyze it, but we
+    don't write about how we will store, manage, organize, and share
+    (even internally with the research group) that data.
+
+-   *Discrepancy between talking about FAIR principles and adhering to
+    FAIR principles.* While there were a few talks on FAIR principles, I
+    didn't see much evidence of them being followed in the presentations
+    or in the projects themselves. This is a very common problem in
+    science (at least in health research), where we talk about the
+    importance of open science, FAIR principles, and reproducibility,
+    but as researchers we don't actually follow them.
+
+-   *Too much focus on genomics, and not other -omic fields.* I was a
+    bit surprised about this, as -omics is a huge field, but a large
+    majority of the presentations and work were in genomics. I had been
+    hoping to learn and hear more about other -omic fields (as I have
+    some experience in lipidomic and metabolomic research), but there
+    was very little of that.
+
+## Interesting insights and ideas
+
+I did get some interesting insights and ideas from the symposium that I
+think might be useful to incorporate or consider in some way in the
+Seedcase Project:
+
+-   There are a lot of -omic based projects, so we need to consider how
+    those types of data will be input into Seedcase software.
+
+-   One presentation was on tracking energy use when running analyses on
+    the server. It made me consider how much energy our work will use
+    and what we could potentially do to lower it.
```

Comment:
Good point. How would we go about that? 
Maybe stuff like:
- Ensuring that the same data isn't added multiple times (deduplication).
- Thinking about if we could compress the data to reduce data size before storing or transmitting (this also includes us thinking about how we currently plan to store the data as both`raw`, `parquet`, *and potentially* SQL).
- Making our UI minimalistic/lightweight by avoiding unnecessary visual elements that require extra processing power.

Reply:
I don't have anything useful to add, but I also find this question very interesting.

Reply:
Yea, I don't really know how to go about it yet either. But it is something to keep in mind as we go along ☺️